runit-ruby/2.3/Dockerfile

```diff
@@ -9,9 +9,11 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get -y update && \
   DEBIAN_FRONTEND=noninteractive apt-get purge -y --auto-remove software-properties-common && \
   rm -rf /var/lib/apt/lists/*
 
+RUN gem update --system
```

Comment:
Can we pin the rubygems and bundler version in this one too?

Reply:
updated